packages/ui-components/src/components/Pagination/Pagination.component.js

```diff
@@ -69,7 +69,8 @@ export const Pagination = ({
 
   const handleNextClick = (event) => {
     // set controlPage +1 if controlPage is not undefined and controlPage is less than controlTotalPage. Set conrolPage +1 also if controlTotalPage is undefined.
-    if (!controlTotalPage || (controlPage && controlPage < controlTotalPage)) {
+    if (controlPage) {
+      if ((controlPage < controlTotalPage) || !controlTotalPage) {
       setControlCurrentPage(controlPage + 1)
```

Comment:
The closing `}` is missing here.
```suggestion
        setControlCurrentPage(controlPage + 1)
      }
```